.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+# Contributing to Strawberry
```

Comment:
It contains some contributing rules that apply to any repo, so it can be moved here. We can mention that "Contributing to code" section is for the main repo.